CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ Bug fixes:
 
 - [#491 Remove redundant Google Analytics](https://github.com/alphagov/govuk_prototype_kit/pull/491)
 
+Unreleased:
+- [#502 Add Cookies and Privacy policy text](https://github.com/alphagov/govuk_prototype_kit/pull/502)
```

Comment:
Do you think this (or perhaps the PR?) should explicitly state that this is for the Kit site (promo mode) rather than the kit in general?